group/edwards25519/point.go

```diff
@@ -290,3 +296,319 @@ func (P *point) IsCanonical(s []byte) bool {
 
 	return 1-(c&d&1) == 1
 }
+
+func (P *point) Hash(m []byte, dst string) kyber.Point {
+	u := hashToField(m, dst, 2)
+	q0 := mapToCurveElligator2Ed25519(u[0])
+	q1 := mapToCurveElligator2Ed25519(u[1])
+	P.Add(q0, q1)
+
+	// Clear cofactor
+	P.Mul(cofactorScalar, P)
+
+	return P
+}
+
+func hashToField(m []byte, dst string, count int) []fieldElement {
+	l := 48 // L param in RFC9380
```

Comment:
I guess this comes from Section 5 in the RFC, right? I'd add a direct pointer to the correct (sub)section.

Reply:
Done !